MaxText/layers/llama2.py

```diff
@@ -127,20 +127,9 @@ def __call__(
     attention_lnx = nn.with_logical_constraint(attention_lnx, activation_axis_names)
     intermediate_inputs = inputs + attention_lnx
 
-    # Fully Connected
-    hidden_states = rms_norm(
-        num_features=intermediate_inputs.shape[-1],
-        dtype=cfg.dtype,
-        weight_dtype=cfg.weight_dtype,
-        name="post_self_attention_layer_norm",
```

Comment:
Does this still work with existing checkpoints? I don't think name is being passed in anymore

Reply:
@phu0ngng Based on our tests, it looks like this does not work properly with existing checkpoints. Can we roll this back or have a roll forward to include `name`?